lang/cs/TestRunner.proj

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Isn't that error prone? I changed files, and ran test project and it ran with old bits? (I might be missing something)